server/app.ts

```diff
@@ -10,7 +10,6 @@ import router from './routes';
 const app = express();
 
 app.use([
-  cors(),
```

Comment:
add the ` cors() ` back plz, I see in a several tutorials that he didn't remove it.